code/espurna/terminal.ino

```diff
@@ -204,7 +204,28 @@ void _terminalInitCommand() {
     #if not SETTINGS_AUTOSAVE
         terminalRegisterCommand(F("SAVE"), [](Embedis* e) {
             eepromCommit();
-            DEBUG_MSG_P(PSTR("\n+OK\n"));
+            terminalOK();
+        });
+    #endif
+
+    #if SECURE_CLIENT == SECURE_CLIENT_BEARSSL
+        terminalRegisterCommand(F("MFLN.PROBE"), [](Embedis* e) {
+            if (e->argc != 3) {
+                terminalError(F("[url] [value]"));
+                return;
+            }
+
+            URL _url(e->argv[1]);
```

Comment:
Should it be `<host> <port> <value>` instead? URL needs to be a bit more clever (or something like external lib instead of bare-minimum parser) to parse things like `example.com:8443`